lesson-2/task-5.js

```diff
@@ -0,0 +1,20 @@
+// Реализовать четыре основные арифметические операции в виде функций с двумя
+// параметрами. Обязательно использовать оператор return .
+
+function addition(a, b) {
+    return a+b;
+}
+
+function subtraction(a, b) {
+    return a-b;
+}
+
+function multiplication(a, b) {
+    return a*b;
+}
+
+function division(a, b) {
+    return a/b;
```

Comment:
Для лучшей читаемости нужно ставить пробелы между операторами и операндами
```suggestion
    return a / b;
```